chapters/en/chapter7/4.mdx

```diff
@@ -222,27 +214,22 @@ print(tokenizer.convert_ids_to_tokens(targets["input_ids"]))
 
 As we can see, using the English tokenizer to preprocess a French sentence results in a lot more tokens, since the tokenizer doesn't know any French words (except those that also appear in the English language, like "discussion").
 
-Both `inputs` and `targets` are dictionaries with our usual keys (input IDs, attention mask, etc.), so the last step is to set a `"labels"` key inside the inputs. We do this in the preprocessing function we will apply on the datasets:
+Since `inputs` is a dictionary with our usual keys (input IDs, attention mask, etc.), the last step is to define the preprocessing function we will apply on the datasets:
 
 ```python
-max_input_length = 128
```

Comment:
Since we use the same length for the inputs and targets, I decided it's simpler to combine them as a single variable